src/Servers/Kestrel/Core/src/Internal/Certificates/ICertificateConfigLoader.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Certificates
+{
+    internal interface ICertificateConfigLoader
```

Comment:
Is this for testing/mocking? 

You could consider using an Abstract Base Class instead (channeling my inner rynowak here) the interface doesn't buy you a lot here since I don't think you plan to have multiple implementations.

Reply:
It's internal and just for testing. Is there some reason using an abstract base class is better for testing?